internal/manager/run.go

```diff
@@ -24,7 +24,9 @@ import (
 	mgrutils "github.com/kong/kubernetes-ingress-controller/v2/internal/manager/utils"
 	"github.com/kong/kubernetes-ingress-controller/v2/internal/util"
 	"github.com/kong/kubernetes-ingress-controller/v2/internal/util/kubernetes/object/status"
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/versions"
 	konghqcomv1 "github.com/kong/kubernetes-ingress-controller/v2/pkg/apis/configuration/v1"
+	konghqcomv1alpha1 "github.com/kong/kubernetes-ingress-controller/v2/pkg/apis/configuration/v1alpha1"
```

Comment:
I suggest to use same pattern to name these packages, either `konghqcom*` or `configuration*`.

Reply:
We can address this in a separate PR via `importas` config  in https://github.com/Kong/gateway-operator/blob/273cd4db2b8e2d8c25dca306db0fa54c21e1bcf5/.golangci.yaml#L47-L62